enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -208,7 +208,8 @@ class AdjointGenerator
         eraseIfUnused(inst);
         if (gutils->isConstantInstruction(&inst))
           return;
-        if (Mode != DerivativeMode::Reverse && Mode != DerivativeMode::Both)
+        if (Mode != DerivativeMode::Reverse && Mode != DerivativeMode::Both &&
```

Comment:
This should probably just be if augmented forward, unless I'm misreading

Reply:
do you mean `DerivativeMode::Forward`?

Reply:
Yeah I mean the existing forward pass in reverse mode. I guess to me "augmented forward" feels more like the "augmented forward pass needed for reverse mode" than "forward mode" so perhaps a name change may be warranted?